internal/command/machine/run.go

```diff
@@ -336,10 +336,18 @@ func parseKVFlag(ctx context.Context, flagName string, initialMap map[string]str
 	return parsed, nil
 }
 
+func safelyGetDockerFilePathFromConfig(cfg *app.Config) string {
```

Comment:
How about making this a method on `*app.Config`, so people could call `cfg.GetDockerfile()` and it will return a string with the path (or an empty string if unset)? I could see this being used in other places.

Reply:
A similar method already existed on appConfig so I just used that.